src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -665,13 +672,13 @@ private BoundExpression BindTupleExpression(TupleExpressionSyntax node, Diagnost
                         Error(diagnostics, ErrorCode.ERR_AnonymousTypeDuplicatePropertyName, argumentSyntax.NameColon.Name);
```

Comment:
Ah... I see. The example would be `(int Name, int Item1, int Item1)` which will complain about both uniqueness and names being reserved. 
I'm not sure what is the better way to report error here. If the user only fixed the third field to `Item2` (solving uniqueness problem), he will still get a reserved name error right afterwards.